plugins/win-capture/inject-helper/CMakeLists.txt

```diff
@@ -5,8 +5,9 @@ if(NOT TARGET OBS::obfuscate)
 endif()
 
 add_library(_inject-helper INTERFACE)
+
 target_sources(_inject-helper INTERFACE inject-helper.c)
-target_compile_options(_inject-helper INTERFACE $<IF:$<CONFIG:Debug>,/MTd,/MT>)
```

Comment:
No longer needed?

Reply:
As per the comment above, I'll change this to the target property `MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>"`.